### DISCUSSION TITLE
CheckList Editor: Add "Remove all checked items" button quickly clean up longer checklists.

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#786] [Feature Requests]
Title says it all

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by kitambi** [comment]
Additional ideas that would help with checklist usability. I didn't feel like opening an entire new issue for this, since they seem related.
* Ability to hide (as opposed to clear) checked items
* Sort unchecked items to the top of the list
* Check off items without opening the full editor (thereby preventing accidentally editing the text)

**Reply by Ventto** [reply]
Same behaviour as Google Keep would be far more convenient. 
No need to clear or remove items. As @kitambi suggested, sort it and add a smarty collapse/expand feature.

**Comment by Ventto** [comment]
@vHanda still ok with the title ?
May I suggest to rename it with something like "Hide (or put aside) checked items for a clean view of unchecked ones" ?
